.github/workflows/append-release-notes.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Append to Release Notes
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v3.3.0
+        with:
+          repository: "trevorwhitney/grafana-github-actions"
+          path: ./actions
+          ref: 0d27c31b3f0c39f6f1f4378aa11b18f9a9fa6c23
+      - name: Install Actions
+        run: npm install --production --prefix ./actions
+      - name: Run release notes appender
+        uses: ./actions/release-notes-appender
+        with:
+          metricsWriteAPIKey: ${{secrets.GRAFANA_MISC_STATS_API_KEY}}
+          token: ${{secrets.GH_BOT_ACCESS_TOKEN}}
+          labelsToAdd: "release-notes"
+          releaseNotesFile: docs/sources/release-notes/v2-8.md
```

Comment:
will update this to `docs/sources/release-notes/next.md` once merged, but that won't exist until this is merged